### PR TITLE
update day values for Kazakh language

### PR DESCRIPTION
**Pull request by talgautb**
I think that's right for day values.
and yes i am native speaker

### CODE REVIEW

**Comment by dkozickis** [comment]
@date-fns/i18n-kk

File: src/locale/kk/_lib/localize/index.js [hunk 1]
```diff
@@ -77,8 +77,8 @@ var formattingMonthValues = {
 
 var dayValues = {
   narrow: ['Ж', 'Д', 'С', 'С', 'Б', 'Ж', 'С'],
-  short: ['жс', 'дс', 'сс', 'ср', 'бс', 'жм', 'сб'],
-  abbreviated: ['жс', 'дс', 'сс', 'ср', 'бс', 'жм', 'сб'],
+  short: ['жк', 'дй', 'сй', 'ср', 'бй', 'жм', 'сн'],
```
Comment (jarjan) [inline src/locale/kk/_lib/localize/index.js]: Can you cite where did you see such short usage of weekdays?

At [Astana Akimat official website](http://astana.gov.kz/) I could find `жс, дс, сс...` usage.

File: src/locale/kk/_lib/localize/index.js [hunk 2]
```diff
@@ -77,8 +77,8 @@ var formattingMonthValues = {
...
+  abbreviated: ['жек', 'дүй', 'сей', 'сәр', 'бей', 'жұм', 'сен'],
```
Comment (jarjan) [inline src/locale/kk/_lib/localize/index.js]: This is good catch!

File: src/locale/kk/_lib/localize/index.js (hunk 1 above)
Comment (talgautb) [reply src/locale/kk/_lib/localize/index.js]: a lot of site use like here https://github.com/moment/moment/blob/develop/src/locale/kk.js

Comment (kossnocorp) [reply src/locale/kk/_lib/localize/index.js]: I see that [CLDR uses](https://www.unicode.org/cldr/charts/32/summary/kk.html#1544) `жс`. Unless there's better standard, we should stick to it.

File: src/locale/kk/_lib/localize/index.js [hunk 3]
```diff
@@ -117,11 +117,11 @@ var formattingDayPeriodValues = {
...
-    noon: 'түс.',
-    morning: 'таң.',
-    afternoon: 'күн.',
-    evening: 'кеш.',
-    night: 'түн.'
+    noon: 'түс',
+    morning: 'таң',
+    afternoon: 'күн',
+    evening: 'кеш',
+    night: 'түн'
```
Comment (kossnocorp) [inline src/locale/kk/_lib/localize/index.js]: @jarjan what do you think about this change?

Comment (kossnocorp) [reply src/locale/kk/_lib/localize/index.js]: cc @drugoi

**Comment by talgautb** [comment]
i have some trouble with windows 10, want to check in mac

File: src/locale/kk/_lib/localize/index.js (hunk 1 above)
Comment (drugoi) [reply src/locale/kk/_lib/localize/index.js]: I agree with @kossnocorp, and I also asked many native kazakh speakers and they confirm, that `жс, дс, сй, etc.` are commonly used format for short date notation.
Please, change it back, @talgautb.

File: src/locale/kk/_lib/localize/index.js (hunk 3 above)
Comment (drugoi) [reply src/locale/kk/_lib/localize/index.js]: Look's good 👍🏼

**Comment by drugoi** [comment]
@talgautb what kind of problem do you have with snapshot generation on Windows 10?

**Comment by talgautb** [comment]
@drugoi i think win problem need another issues,
ok i will check values

**Pull request merged**